extensions/sql-bindings/src/common/utils.ts

```diff
@@ -162,6 +162,28 @@ export function escapeClosingBrackets(str: string): string {
 	return str.replace(']', ']]');
 }
 
+/**
+ * Gets a unique object name by removing all special characters
+ * @param objectName can include brackets/periods and user entered special characters
+ * @returns the object name without any special characters
+ */
+export function getUniqueObjectName(objectName: string): string {
```

Comment:
There's nothing about this that's making it "unique". I would instead call this `sanitizeObjectName`